packages/babel-register/README.md

```diff
@@ -68,7 +68,10 @@ require("@babel/register")({
   extensions: [".es6", ".es", ".jsx", ".js", ".mjs"],
 
   // Setting this to false will disable the cache.
-  cache: true
+  cache: true,
+
+  // Specify the version of babel-core used for transformation. (Optional)
+  babel: require('babel-core')
```

Comment:
`require("@babel/core")`?

Reply:
@nicolo-ribaudo Imagine calling `babel-register` from a location where a different version of `@babel/core` is installed or available than the one that is bundled with `babel-register`

Reply:
That's a good point. `@babel/register` should also work with `@babel/core` and same for babel-register and babel-core.

Reply:
Well, in my case, it was using `@babel/core` but a conflicting version.

Reply:
Could you please change `babel-core` to `@babel/core`, then I think we're ready to merge

Reply:
Done